### ISSUE TITLE
Add my signature

### ISSUE THREAD

**Issue by larson-carter** [drop-ice/dear-github-2.0#876]
Hi, I’d like to sign this letter in support of ending GitHub’s business with ICE. Please add my name, GitHub username, and affiliations as shown below.

**Name: Larson Carter, @larson-carter** 
**Affiliations (optional): OhMyZsh** 

<!--
(Examples of affiliations are organizations you work for or with, projects you contribute to, and/or anything else that feels appropriate for you.)
--!>

**Comment by madilraza** [comment]
Do yourself

**Comment by larson-carter** [comment]
@madilraza No. The purpose of issues is for people to add it for you. I'm too busy to draft it myself. However you are welcome to do it for me!

**Comment by donno2048** [comment]
View at #968

**Comment by larson-carter** [comment]
@donno2048 Thank you very much!

**Comment by donno2048** [comment]
You're welcome 😊
